Makefile

```diff
@@ -56,20 +63,24 @@ clippy: install_rs_check_toolchain
 .PHONY: build
 build: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release
+		--features=$(FFT128_FEATURE)
 
 .PHONY: build_no_std
 build_no_std: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release \
 		--no-default-features
+		--features=$(FFT128_FEATURE)
 
 .PHONY: build_bench
 build_bench: install_rs_check_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_CHECK_TOOLCHAIN) bench \
 		--no-run
+		--features=$(FFT128_FEATURE)
 
 .PHONY: test
 test: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --release
+		--features=$(FFT128_FEATURE)
```

Comment:
not sure rust accepts empty feature, also missing \ on the previous line for a line break

Reply:
cargo accepts empty features.

Reply:
ok for the empty feature then